Src/AutoFixture.NUnit3.UnitTest/PilotTest.cs

```diff
@@ -0,0 +1,14 @@
+﻿using NUnit.Framework;
+
+namespace Ploeh.AutoFixture.NUnit3.UnitTest
+{
+    [TestFixture]
+    public class PilotTest
+    {
+        [Test]
+        public void AssertTrue()
+        {
+            Assert.IsTrue(true);
```

Comment:
No, it's here because there is nothing else to write a test against to show that the project is set up properly. So it should disappear with the appearance first meaningful unit test.